cmd/influxd/run/config.go

```diff
@@ -79,6 +85,14 @@ func NewConfig() *Config {
 // NewDemoConfig returns the config that runs when no config is specified.
 func NewDemoConfig() (*Config, error) {
 	c := NewConfig()
+	// START https://github.com/influxdb/influxdb/issues/4752
```

Comment:
`START` and `END` are not something we normally do. A simple one-liner linking the github issue would be better.